otaclient/app/ecu_info.py

```diff
@@ -61,13 +61,14 @@ class ECUInfo:
     def parse_ecu_info(cls, ecu_info_file: Union[str, Path]) -> "ECUInfo":
         ecu_info = deepcopy(DEFAULT_ECU_INFO)
         try:
-            ecu_info = yaml.safe_load(Path(ecu_info_file).read_text())
+            _ecu_info = yaml.safe_load(Path(ecu_info_file).read_text())
             assert isinstance(ecu_info, Dict)
```

Comment:
asserting `ecu_info` at this line is strange. Can you add test for this PR?

Reply:
Sure! This line is to prevent the case that ecu_info.yaml is valid yaml but not valid ecu_info file(like top level is array, empty file, etc.).